napari/components/viewer_model.py

```diff
@@ -926,6 +926,116 @@ def add_surface(
         self.add_layer(layer)
         return layer
 
+    def add_swc(
+        self,
+        swc_path,
+        spacing,
+        origin=np.array([0, 0, 0]),
+        *,
```

Comment:
What does this `*` do?

Reply:
`*` is to specify that every argument before it is a positional argument and every after argument after is a keyword argument

https://www.python.org/dev/peps/pep-3102/